pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -81,179 +89,296 @@ def is_float(istring):
         return 0
 
 
-class PostProcessor(Post):
-    """Contains advanced postprocessing functionalities that require Python 3.x packages like NumPy and Matplotlib.
-
-    Parameters
-    ----------
-    app :
-        Inherited parent object.
+class ObjClass(object):
+    """Class that manages mesh files to be plotted in pyvista."""
 
-    Examples
-    --------
-    Basic usage demonstrated with an HFSS, Maxwell, or any other design:
+    def __init__(self, path, color, opacity, units):
+        self.path = path
+        self._color = (0, 0, 0)
+        self.color = color
```

Comment:
I think that you can only set the protected attribute. Then the color property will return the correct value latter on.
```suggestion
        self._color = color
```

Reply:
the color setter is handling both a tuple or a string "white". That's why you have to use the setter and not instantiate directly the internal variable

Reply:
ok, got it.